bower.json

```diff
@@ -30,16 +30,19 @@
     "angular": "~1.3.0",
     "angular-bootstrap": "~0.11.1",
     "angular-bootstrap-slider": "~0.0.6",
+    "angular-load": "~0.4.1",
     "angular-sanitize": "~1.3.0",
     "angular-ui-router": "0.2.15",
-    "rv-angular-bootstrap-colorpicker": "https://github.com/Rise-Vision/angular-bootstrap-colorpicker.git",
-    "rv-common-style": "https://github.com/Rise-Vision/common-style.git",
+    "angular-ui-tinymce": "~0.0.12",
+    "bootstrap-form-components": "https://github.com/Rise-Vision/bootstrap-form-components.git",
+    "bootstrap-sass-official": "~3.2.0",
     "component-storage-selector": "https://github.com/Rise-Vision/component-storage-selector.git",
     "component-subscription-status": "https://github.com/Rise-Vision/component-subscription-status.git",
-    "bootstrap-sass-official": "~3.2.0",
-    "bootstrap-form-components": "https://github.com/Rise-Vision/bootstrap-form-components.git",
+    "rv-angular-bootstrap-colorpicker": "https://github.com/Rise-Vision/angular-bootstrap-colorpicker.git",
+    "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git",
+    "rv-common-style": "https://github.com/Rise-Vision/common-style.git#1.3.47",
     "spectrum": "https://github.com/Rise-Vision/spectrum.git",
-    "rv-common-i18n": "https://github.com/Rise-Vision/common-i18n.git"
+    "tinymce": "~4.3.3"
```

Comment:
@donnapep The `tinymce` package already gets brought in via `angular-ui-tinymce`, you can safely omit specifying it here.